src/apiClients/AirTableApiClient.ts

```diff
@@ -24,9 +35,61 @@ class AirTableApiClient {
       `interestCategories: ${interestCategories}`,
       `riverSections: ${riverSections}`,
     );
+    let organizations: Organization[] = [];
+
+    const promise: Promise<Organization[]> = new Promise((resolve, reject) => {
+      this.base('Organizations')
+        .select({
+          view: 'Grid view',
+        })
+        .eachPage(
+          (records: any[], fetchNextPage: any) => {
+            let filteredRecords: any[] = records;
+            if (typeof riverSections !== 'undefined') {
+              filteredRecords = records.filter(record => {
+                return riverSections.includes(record.get('River Section'));
+              });
+            }
+            if (typeof interestCategories !== 'undefined') {
+              filteredRecords = filteredRecords.filter(record => {
+                return (
+                  interestCategories.filter(category => {
+                    return record.get('Interest Categories').includes(category);
+                  }).length != 0
+                );
+              });
+            }
+            organizations = organizations.concat(
+              filteredRecords.map((record: any) => {
+                if (typeof record === 'undefined') {
+                }
```

Comment:
why is this here? (although this entire `.map` call should disappear if you implement the advice above)